operations/helm/charts/mimir-distributed/templates/_helpers.tpl

```diff
@@ -617,3 +617,19 @@ mimir.siToBytes takes 1 argument
         {{- .value }}
     {{- end -}}
 {{- end -}}
+
+{{/*
+parseCPU is used to convert Kubernetes CPU units to the corresponding float value of CPU cores.
+The returned value is a string representation. If you need to do any math on it, please
```

Comment:
This seems unfinished.
```suggestion
The returned value is a string representation. If you need to do any math on it, please parse the string first.
```

Did you mean this?

Reply:
Ops, yes.